### PR TITLE
LedgerApiTestTool prints summary of skipped (excluded) tests [DPP-538]

### PR DESCRIPTION
**Pull request by tudor-da**
LedgerApiTestTool prints list of excluded test at the end of the report summary.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by tudor-da** [comment]
image

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Result.scala [hunk 1]
```diff
@@ -12,6 +12,7 @@ private[daml] object Result {
 
   final case class Succeeded(duration: Duration) extends Success
   case object Retired extends RuntimeException with NoStackTrace with Success
+  case object Skipped extends RuntimeException with NoStackTrace with Success
```
Comment (kamil-da) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Result.scala:15]: How about using `Excluded` consistently?

Comment (tudor-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Result.scala]: good observation, changed

**Review by kamil-da — APPROVED** [review]

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ParticipantPruningIT.scala [hunk 2]
```diff
@@ -628,9 +628,12 @@ class ParticipantPruningIT extends LedgerTestSuite {
     } yield ()
   })
 
+  // This test can only be run in multi-participant setups, since the participant
+  // on which we assert pruning of the immediate divulgence
+  // must not have a locally-hosted stakeholder of the divulged contract.
   test(
-    "PRDisclosureAndRetroactiveDivulgence",
-    "Disclosure pruning succeeds",
+    "PRImmediateAndRetroactiveDivulgence",
```
Comment (tudor-da) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ParticipantPruningIT.scala:635]: Renamed for consistency

**Review by mziolekda — COMMENTED** [review]
Nice work. I have seen it in action this morning. It's very helpful to know the excluded tests

**Pull request merged**